apps/ephemeral/specs/index.spec.tsx

```diff
@@ -13,4 +13,9 @@ describe('Index', () => {
     const { getByText } = render(<Index />);
     expect(getByText('Welcome to ephemeral!')).toBeTruthy();
   });
+
+  it('Resources & Tools should render on the page', () => {
```

Comment:
Would you consider this a unit test or an integration test?

![image](https://user-images.githubusercontent.com/210413/69075608-52444780-09f7-11ea-8ee4-ccd6a7a30fe8.png)

Reply:
Since it's all a part of a single component, I would consider it to be a unit test. If I were testing the interaction between two components, that would be an integration test.